stories/breadcrumb.stories.js

```diff
@@ -135,7 +135,7 @@ storiesOf('Breadcrumb', module)
       search.addWidget(
         instantsearch.widgets.breadcrumb({
           container: breadcrumb,
-          separator: ' / ',
+          separator: ' > ',
```

Comment:
The option is really useful? Did you tried without it?

Reply:
correct, removed altogether.